app/views/items/destroy.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if @item.destroyed? %>
```

Comment:
I am not sure why the JS is not working but thought maybe it has something to do with this .destroyed? - am I destroying the item and then running this causing some confusion? Not sure what is happening...